src/components/card.js

```diff
@@ -26,12 +26,45 @@ getTopRatedFilms().then(result => {
 
     filmImg.addEventListener('click', showDetailingInfoAboutFilm);
 
-});
+}); */
 
-function showDetailingInfoAboutFilm() {
+/* function showDetailingInfoAboutFilm() {
     history.pushState({page: 1}, "title 1", "?page=1");
     let divAboutFilm = document.createElement('div');
     let backGroundImage = 
 
     divAboutFilm.style.zIndex = '1000';
+} */
+
+export function createMovieCard() {
```

Comment:
@Mmarvinn Task wasn't about  `MovieCard`, why did you put it in this PR?

Reply:
sorry, my bad.